script/build-wasm

```diff
@@ -77,6 +77,8 @@ fi
 
 mkdir -p target/scratch
 
+runtime_methods='stringToUTF16','AsciiToString'
```

Comment:
Next, I encountered this error:
![Screenshot from 2022-10-16 18-17-01](https://user-images.githubusercontent.com/2058614/196061781-21c16426-b90f-4df1-9720-1f94fdb5e394.png)

After some googling I discovered that [`stringToUTF16` is from emscripten's preamble](https://emscripten.org/docs/api_reference/preamble.js.html#stringToUTF16) and just needed exporting.

Next, the line `console.log(tree.rootNode.toString());` threw this error:
![Screenshot from 2022-10-16 18-25-15](https://user-images.githubusercontent.com/2058614/196061840-3527ad37-0d3f-469d-aed7-85e517b4be44.png)

I quickly determined that the cause is the same and added it as well.